### ISSUE TITLE
child experiment tracker

### ISSUE THREAD

**Issue by leondz** [Breakend/experiment-impact-tracker#10]
In multi-experiment code it would be constructive to launch one tracker and store child experiments within this. There seems to be some support for this but it's unclear how to use it - is this feature implemented?

**Comment by Breakend** [comment]
I'm not sure I understand this use case or whether it would even be possible. We use the process ids for the script to attribute energy. We can't attribute energy to threads within python or multiple experiments within one process. I think the recommended approach would be to use multiple scripts each with their own dedicated tracker

**Comment by leondz** [comment]
Ah, OK. This was a reference to this part of the example in the root README.md. My use case is to have an experiment runner that measures impact of multiple experiments. 

```
# The child experiments to group together
  "child_experiments" : 
```

**Comment by Breakend** [comment]
Might be addressed by #24 ? We now support contexts and have tests to validate. Please re-open if still an issue.
